src/collective/easyform/browser/actions.py

```diff
@@ -244,8 +244,8 @@ class EasyFormActionsListingPage(SchemaListingPage):
 
 
 class ActionAddForm(FieldAddForm):
-
-    fields = field.Fields(INewAction)
+    schema = INewAction
+    fields = field.Fields()  # let fields be updated by AutoExtensibleForm.updateFields
```

Comment:
Do we actually need to initialize the fields attribute? *Maybe* not setting it at all works as well.

Reply:
Yes, we do. plone.schemaeditor explicitly sets the fields wich makes any autoform/supermodel directives ineffective. That should probably be fixed there in the first place but I don't want to introduce a version dependency here.
Re-setting the fields like that allows autoform fill out the fields from the schema and consider the directives also...